### PR TITLE
add SIG Security annual report

### PR DESCRIPTION
**Pull request by IanColdwater**
**Which issue(s) this PR fixes**:

Adding annual report template for SIG Security

### CODE REVIEW

File: sig-security/annual-report-2021.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    *   In addition to participating in SIG discussions, new contributors can also begin by assisting the subprojects with their tasks. Subproject members guide new contributors to where assistance is needed.
+    *   Our open environment creates space for interested contributors to become leaders [by sharing their ideas and energy](https://twitter.com/PuDiJoglekar/status/1380205701994147845). If someone brings an idea that other members are excited about and want to help work on, [those members can organize a subproject together](https://twitter.com/coffeeartgirl/status/1331046904306622465), and we can help them grow. 
+    *   The chairs help grow emerging leaders by keeping in close contact with them and providing 1:1 mentorship, [encouragement](https://twitter.com/coffeeartgirl/status/1337078959020912642), and concrete help leveling up skills such as improving communication, leading more inclusive meetings, and presenting at conferences.
```
Comment (mrbobbytables) [inline sig-security/annual-report-2021.md:46]: This is great :)
cc @savitharaghunathan

File: sig-security/annual-report-2021.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    *   KEP [2568](https://github.com/kubernetes/enhancements/issues/2568) - Run control-plane as non-root in kubeadm - has been maturing steadily with help from the Kubernetes community. SIG Security has provided a forum to discuss it and feedback to advance it toward implementable status.
+    *   KEP [1981](https://github.com/kubernetes/enhancements/issues/1981) - Support for Windows privileged containers - has matured toward alpha, including SIG Security member feedback. Nice work, team!
+*   **What areas and/or subprojects does the group need the most help with?**
+    *   We always welcome help from anyone aligned with our vision of improving Kubernetes security by learning together, sharing our expertise, and encouraging cross-SIG collaboration. Ideally, we would like to eventually have participating members with crossover to every SIG, providing a natural conduit of ideas and forming a network of helpful hackers throughout Kubernetes.
```
Comment (mrbobbytables) [inline sig-security/annual-report-2021.md:66]: Long term a codified liaison program might help. Someone from sig-security that gets involved in another sig they have some expertise in that can be involved in triage sessions or pinged with questions / potentially escalate.

File: sig-security/annual-report-2021.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            *   Come to our meetings and get more involved! If you have friends who are interested in Kubernetes security, encourage them to come and bring their energy and ideas too. We’d love to see you and hear what you have to say!
+        *   To bring additional expertise to the table and help balance the load for SIG Leads, we would like to grow new Leads in the future, and are determining how best to implement the Tech Lead role in a SIG like ours. We would like to grow one or two interested members into that new role, and will be looking for help with this bootstrapping process when that time comes.
+*   **What's the average open days of a PR and Issue in your group?**
+    *   As a SIG who doesn’t own code in a project that encourages non-code contributions, we find the question of which PRs and issues belong to us to be difficult to answer, and would like to encourage further inclusivity of different kinds of contribution in questions on future reports.
```
Comment (mrbobbytables) [inline sig-security/annual-report-2021.md:75]: I can relate as someone that's involved in a significant amount of non-code contributions. The way we (ContribEx) try and gain greater visibility into this is by moving AIs and tasks into issues as that is broadly what the community uses and the tooling supports. This is largely for greater discoverability, recognition and transparency (both as a project and for employers who look for this sort of thing 😬).

One potential thought, there are several SIGs that have their own dedicated repos (e.g. [sig-release](http://github.com/kubernetes/sig-release) and [sig-testing](http://github.com/kubernetes/sig-testing)) that serve as "meta" repos for documentation on processes and a place where they can create and assign issues.

I'd be happy to chat about potential options both GitHub centric and other ideas. :) We've explored a few that could bubble other metrics like slack/mailing list activity, but so far they all get into the creepy area of PII and lack of [consent](https://github.com/kubernetes/community/issues/5260#issuecomment-763037701) >_>

**Review by mrbobbytables — COMMENTED** [review]
Thank you! This report looks great. :)
No blockers, just a few ideas and thoughts \o/

File: sig-security/annual-report-2021.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    *   We measure our reviewer bandwidth by noting whether everything brought to SIG Security’s attention is getting addressed in a timely fashion, and the feelings of the reviewers about their workload.
+    *   Much of this review is currently done by the SIG chairs, and it is starting to become a bottleneck. We are looking to improve our review bandwidth by bringing review of triaged issues to the regular SIG meetings for more open and inclusive participation. 
+*   **Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+    *   We are onboarding and growing contributors continuously through our processes and culture. We actively practice behaviors of inclusion to help new and growing contributors feel more confident to participate. For example, we list appropriate pronouns in our meeting attendance list, and before changing subjects we pause to ask for additional comments.
```
Comment (parispittman) [inline sig-security/annual-report-2021.md:43]: this is not a blocking question, just discovery - how are you doing outreach for new folks to come on board?

File: sig-security/annual-report-2021.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    *   The chairs help grow emerging leaders by keeping in close contact with them and providing 1:1 mentorship, [encouragement](https://twitter.com/coffeeartgirl/status/1337078959020912642), and concrete help leveling up skills such as improving communication, leading more inclusive meetings, and presenting at conferences.
+*   **What programs do you participate in for new contributors?**
+    *   Our broadly welcoming culture is a good opportunity for security-minded Kubernetes users who want to get involved in the upstream Kubernetes community. We plan to extend this further by participating in formal “new contributor” programs in the future.
+*   **Does the group have contributors from multiple companies/affiliations?**
+    *   Yes
+    *   **Can end users/companies contribute in some way that they currently are not?**
+        *   End users are currently contributing at all levels. Our contributors represent both end user organizations and Kubernetes vendors. Both SIG Security co-chairs work at companies that are Kubernetes end users.
```
Comment (parispittman) [inline sig-security/annual-report-2021.md:52]: if you had a microphone to promote sig-security to all end users, what would you say you need the most help with or how you'd like them to interact with the group?

File: sig-security/annual-report-2021.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Current initiatives and project health
+
+*   **What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
```
Comment (parispittman) [inline sig-security/annual-report-2021.md:56]: thank you for this level of detail with the initiatives

**Comment by parispittman** [comment]
no blockers for me, mostly discovery questions and commentary

**Pull request merged**